src/main/docker/DISABLE-CBC.pmod

```diff
@@ -0,0 +1 @@
+cipher = -AES-128-CBC -AES-256-CBC
```

Comment:
Is there a syntax that would allow us to have a copyright header in this file?

I also notice warnings in the build log due to this so we should either add the appropriate mapping or if there is none then add an exclusion.
```
2022-08-23T21:09:42.766514354Z [WARNING] Unknown file extension: /wd/src/main/docker/DISABLE-CBC.pmod
2022-08-23T21:09:42.779015285Z [WARNING] Unable to find a comment style definition for some files. You may want to add a custom mapping for the relevant file extensions.
```